aleph/manage.py

```diff
@@ -82,7 +85,28 @@ def ensure_collection(foreign_id, label):
     return collection
 
 
-@click.group(cls=FlaskGroup, create_app=create_app)
+def get_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
```

Comment:
A very minor thing, but I find this name better describes what this function does:

```suggestion
def print_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
```

Reply:
Agreed and changed.

Reply:
Thanks!